src/modules/ImageSelect.js

```diff
@@ -67,7 +67,7 @@ module.exports = function ImageSelect(options) {
 
   // this module is unique because it creates the image
   function draw(image) {
-    options.el.html(image);
+    el.html(image);
```

Comment:
Do you mean to include these changes, or is this new commit based on something other than the latest `master` branch? Thanks!